src/libmongoc/doc/mongoc_collection_find_indexes.rst

```diff
@@ -37,4 +37,9 @@ Errors are propagated via the ``error`` parameter.
 Returns
 -------
 
-A cursor where each result corresponds to the server's representation of an index on this collection. If the collection does not exist on the server, the cursor will be empty.
+.. include:: includes/returns-cursor.txt
+
+In the returned cursor each result corresponds to the server's representation of an index on this collection. If the collection does not exist on the server, the cursor will be empty.
+
+The cursor functions :symbol:`mongoc_cursor_set_limit`, :symbol:`mongoc_cursor_set_batch_size`, and :symbol:`mongoc_cursor_set_max_await_time_ms` have no use on the returned cursor.
```

Comment:
I see a similar paragraph in mongoc_collection_find_indexes_with_opts.rst and I have included it under the understanding that both functions behave the same.

However, if I'm wrong, just tell me and I'll remove the paragraph.

Reply:
Good catch! This is an improvement. `mongoc_collection_find_indexes` directly calls `mongoc_collection_find_indexes_with_opts`, so the same behavior applies to the returned cursor.